### PR TITLE
Minor fixes for CI updater shell script.

### PR DESCRIPTION
**Pull request by RubyNova**
Title.

### CODE REVIEW

File: ci-scripts/update-all-on-remote.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
+set -e
```
Comment (Banananaman91) [inline ci-scripts/update-all-on-remote.sh:1]: Is this still needed?

File: ci-scripts/update-all-on-remote.sh [hunk 2]
```diff
@@ -11,11 +13,11 @@ case $input in
...
             newBranch=${branch#"$originPrefix"}
 
-            if $newBranch != "master"; then
+            if [ "${newBranch}" != "master" ]; then
                 echo "INFO: Attempting rebase of ${branch}"
-                git checkout $newBranch
+                git checkout "${newBranch}"
                 git rebase development
-                git push
+                #git push
```
Comment (Banananaman91) [inline ci-scripts/update-all-on-remote.sh]: Can you uncomment this

**Review by Banananaman91 — CHANGES_REQUESTED** [review]

File: ci-scripts/update-all-on-remote.sh (hunk 2 above)
Comment (RubyNova) [reply ci-scripts/update-all-on-remote.sh]: Woops. Sure.

File: ci-scripts/update-all-on-remote.sh (hunk 1 above)
Comment (RubyNova) [reply ci-scripts/update-all-on-remote.sh:1]: I'm fairly certain it is, from what I was seeing. Its a hotfix for early exit when something fails.

**Review by Banananaman91 — APPROVED** [review]

**Pull request merged**